irohad/torii/processor/query_processor_impl.hpp

```diff
@@ -44,8 +44,8 @@ namespace iroha {
        * Register client query
        * @param query - client intent
        */
-      void queryHandle(
-          std::shared_ptr<shared_model::interface::Query> qry) override;
+      std::unique_ptr<shared_model::interface::QueryResponse> queryHandle(
+          const shared_model::interface::Query &qry) override;
 
       /**
```

Comment:
Maybe remove comments in this file? They do not add anything new to their ancestors in interface